pkg/trace/stats/concentrator_test.go

```diff
@@ -208,10 +208,10 @@ func TestConcentratorOldestTs(t *testing.T) {
 	t.Run("hot", func(t *testing.T) {
 		flushTime := now.UnixNano()
 		c := NewTestConcentrator(now)
-		c.oldestTs = alignTs(flushTime, c.bsize) - int64(c.bufferLen-1)*c.bsize
+		c.spanConcentrator.oldestTs = alignTs(flushTime, c.bsize) - int64(c.spanConcentrator.bufferLen-1)*c.bsize
```

Comment:
Can we get away with just having `SpanConcentrator` encapsulate `bsize` rather than also being in the `Concentrator` struct (seems only used in these tests)?